### PR TITLE
Post information show from DB

### PR DESCRIPTION
**Pull request by Javi1499**
# Description
What did you implemented/Why did you implemented this:
 - Static information in post view was resplaced for DB information for each post
 - Carousel in post view was added to show all images 
- Average of comments is dynamic depending all comments and ranking
- 
   ## Screenshots  
May-17-2022 18-52-50

  ## Screenshots

### CODE REVIEW

**Review by germanruzca — DISMISSED** [review]
LGTM

**Review by nallely64 — APPROVED** [review]
LGTM!

File: app/assets/stylesheets/components/_cards.scss [hunk 1]
```diff
@@ -20,7 +20,7 @@
   &__image {
     display: flex;
     justify-content: center;
-    width: 320px;
+    width: 100%;
     height: 300px;
```
Comment (JuanequeX) [inline app/assets/stylesheets/components/_cards.scss:24]: Sort alphabetically this block

File: app/assets/stylesheets/components/_image.scss [hunk 2]
```diff
@@ -1,4 +1,4 @@
-.image{
+.image {
   width: 100%;
   height: 100%;
```
Comment (JuanequeX) [inline app/assets/stylesheets/components/_image.scss:3]: Sort alphabetically this block

File: app/assets/stylesheets/components/_post_form.scss [hunk 3]
```diff
@@ -15,25 +15,53 @@
...
     flex-direction: column;
     margin: 20px 0;
+
+    &_radio {
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+
+      label:first-child {
+        font-family: 'Roboto';
```
Comment (JuanequeX) [inline app/assets/stylesheets/components/_post_form.scss:36]: Will be better use a variable to add a font-family, what do you think?

File: app/assets/stylesheets/pages/_post.scss [hunk 4]
```diff
@@ -1,24 +1,29 @@
...
+      "reviews" auto/
+      100%;
   }
 
   &__image {
     grid-area: image;
+    height: 90%;
+    width: 90%;
+    display: flex;
+    justify-content: center;
```
Comment (JuanequeX) [inline app/assets/stylesheets/pages/_post.scss:26]: Sort alphabetically this block.

**Review by JuanequeX — CHANGES_REQUESTED** [review]

**Review by JuanequeX — APPROVED** [review]
LGTM

**Review by germanruzca — APPROVED** [review]
LGTM

**Pull request merged**